waltz-data/src/main/java/org/finos/waltz/data/logical_flow/LogicalFlowDao.java

```diff
@@ -413,6 +425,7 @@ private SelectJoinStep<Record> baseQuery() {
         return dsl
                 .select(LOGICAL_FLOW.fields())
                 .select(SOURCE_NAME_FIELD, TARGET_NAME_FIELD)
+                .select(SOURCE_EXTERNAL_ID_FIELD, TARGET_EXTERNAL_ID_FIELD)
```

Comment:
We'll need to check this in prod to make sure it doesn't degrade the qry performance too much

Reply:
Agreed